PlayCover/Model/PlayApp.swift

```diff
@@ -86,7 +86,7 @@ class PlayApp: BaseApp {
                             &assertionID)
                         if success == kIOReturnSuccess {
                             while true { // Run a loop until the app closes
-                                Thread.sleep(forTimeInterval: 10) // Wait 10s
+                                try await Task.sleep(nanoseconds: 10000000000) // Sleep for 10 seconds
```

Comment:
`try await Task.sleep(for: .second(10))`

Reply:
`sleep(for:)` is only available in macOS 13, as other suggestions.

Reply:
As Amir said, `sleep(for:)` is only available on macOS 13+